googlepay/src/main/java/com/adyen/checkout/googlepay/GooglePayComponent.java

```diff
@@ -87,12 +87,7 @@ private GooglePayParams getGooglePayParams() {
         final Configuration configuration = ((GenericPaymentMethodDelegate) mPaymentMethodDelegate)
                 .getPaymentMethod()
                 .getConfiguration();
-        final String serverGatewayMerchantId;
-        if (configuration == null) {
-            serverGatewayMerchantId = null;
-        } else {
-            serverGatewayMerchantId = configuration.getGatewayMerchantId();
-        }
+        final String serverGatewayMerchantId = (configuration != null) ? configuration.getGatewayMerchantId() : null;
```

Comment:
HAHAHAHAHAHA, I agree but I think this and the Release notes thing should be it's own PR